presto-docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -34,13 +34,64 @@ Presto restart. The refresh period is specified in the ``etc/access-control.prop
+access to catalogs, but does not explicitly grant any specific schema or table permissions.
+The table and schema rules are used to specify who can can create, drop, alter, select, insert,
+delete, etc. for schemas, tables and views.
+
+For each rule set, permission is based on the first matching rule read from top to bottom.  If
+no rule matches, access is denied. If no rules are provided at all, then access is granted.
+
+The following table summarizes the permissions required for each SQL command:
+
+==================================== ========== ======= ==================== ===================================================
+SQL Command                          Catalog    Schema  Table                Note
+==================================== ========== ======= ==================== ===================================================
+SHOW CATALOGS                                                                Always allowed
+SHOW SCHEMAS                         read-only  any*    any*                 Allowed if catalog is :ref:`visible<visibility>`
+SHOW TABLES                          read-only  any*    any*                 Allowed if schema :ref:`visible<visibility>`
+CREATE SCHEMA                        read-only  owner
+DROP SCHEMA                          all        owner
+SHOW CREATE SCHEMA                   all        owner
+ALTER SCHEMA ... RENAME TO           all        owner*                       Ownership is required on both old and new schemas
+ALTER SCHEMA ... SET AUTHORIZATION   all        owner
+CREATE TABLE                         all                owner
+DROP TABLE                           all                owner
+ALTER TABLE ... RENAME TO            all                owner*               Ownership is required on both old and new tables
+CREATE VIEW                          all                owner
+DROP VIEW                            all                owner
+ALTER VIEW ... RENAME TO             all                owner*               Ownership is required on both old and new views
+COMMENT ON TABLE                     all                owner
+COMMENT ON COLUMN                    all                owner
+ALTER TABLE ... ADD COLUMN           all                owner
+ALTER TABLE ... DROP COLUMN          all                owner
+ALTER TABLE ... RENAME COLUMN        all                owner
+SHOW COLUMNS                         all                any
+SELECT FROM table                    read-only          select
+SELECT FROM view                     read-only          select, grant_select
+INSERT INTO                          all                insert
+DELETE FROM                          all                delete
+==================================== ========== ======= ==================== ===================================================
+
+.. _visibility:
+
+For a catalog, schema, or table to be visible in a ``SHOW`` command, the user must have
```

Comment:
maybe have an actual section header Visibility ?

Reply:
I was thinking the section headers would be reserved for the rules sections, but a header seems ok to me